types/dhcppool.pp

```diff
@@ -0,0 +1,7 @@
+type Dhcp::DhcpPool = Struct[
```

Comment:
It would be nice to add some tests using https://rspec-puppet.com/documentation/type_aliases/ and also add some comment so puppet-strings can generate something useful.